RELEASE.md

```diff
@@ -144,11 +155,11 @@ Even though this release ships a fix for project generated with `kedro==0.16.2`,
 ### DataSets
 * Added the following new datasets.
 
-| Type                       | Description                                 | Location                                         |
-| -------------------------- | ------------------------------------------- | ------------------------------------------------ |
-| `pillow.ImageDataSet`      | Work with image files using `Pillow`        | `kedro.extras.datasets.pillow`                   |
-| `geopandas.GeoJSONDataSet` | Work with geospatial data using `GeoPandas` | `kedro.extras.datasets.geopandas.GeoJSONDataSet` |
-| `api.APIDataSet`           | Work with data from HTTP(S) API requests    | `kedro.extras.datasets.api.APIDataSet`           |
+| Type             | Description                                 | Location                          |
+| ---------------- | ------------------------------------------- | --------------------------------- |
+| `ImageDataSet`   | Work with image files using `Pillow`        | `kedro.extras.datasets.pillow`    |
```

Comment:
I think that's fine, I just think it should be standardized through the document. :) I took what I felt was most common. I can change everything if that's preferred.

Reply:
I think the general rule of thumb is that we can omit the common prefix, but we need to keep the expression that the users can put into their `catalog.yml` and rest assured that it will work 🙂

Reply:
@DmitriiDeriabinQB Mind checking if this is acceptably resolved on the public PR?  I left locations as is, and made types into the path that users can put into `catalog.yml` (e.g. api.APIDataSet`).